internal/tfimport/importer/simple_defaults_importer.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package importer
+
+import (
+	"github.com/GoogleCloudPlatform/healthcare-data-protection-suite/internal/terraform"
+)
+
+// SimpleDefaultsImporter defines a struct with the necessary information to import a resource that only depends on fields from the plan.
```

Comment:
I don't quite understand why this needs to be a separate resource. We can't just add a Defaults field under SimpleImporter?

Reply:
I thought that every other resource would need to provide an empty Defaults. But now I realize that if not provided, it will just be the empty value. I'll adjust the code.